integration_tests/test_basic_routes.py

```diff
@@ -0,0 +1,73 @@
+# Test the routes with:
```

Comment:
Do you think something like Parameterizing will be helpful here(https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids) ?

Reply:
Much better indeed, I struggled to make it work with fixtures but it is done now, I will push it soon :slightly_smiling_face: